i18n/en.json

```
@@ -1121,6 +1121,9 @@
   "admin.license.keyRemove": "Remove Enterprise License and Downgrade Server",
   "admin.license.noFile": "No file uploaded",
   "admin.license.removing": "Removing License...",
+  "admin.license.subscription.privateCloudCard.description": "Review your numbers below to ensure you renew for theright number of users.",
```

Given that this is not related to Cloud I would change the naming to something like (including the couple below: 

```suggestion
  "admin.license.renewalCard.description": "Review your numbers below to ensure you renew for the right number of users.",
```
(There was also a space missing)